tests/main.nf.test

```diff
@@ -4,6 +4,7 @@ nextflow_pipeline {
     script "../main.nf"
     profile "test"
     tag "PIPELINE"
+    config "../conf/test.config"
```

Comment:
Should this be `"tests/nextflow.config"` ?

Reply:
No, I think (although I'm not 100% sure) that now I have a `tests/nexflow.config` that is active as the "base" nf-test config for all tests (including the subworkflows) which do not contain any input files. 

For the pipeline tests in `tests/main.nf.test` I want to overwrite the "base" nf-test config with the "normal" test profile config in conf/test.config, to use the input files and parameters specified there.

A bit of trial and error 😅